### ISSUE TITLE
Notification if Transcoding

### ISSUE THREAD

**Issue by Maxplosion** [dkanada/gelli#71]
It would be nice to have some kind of visual notification if the song is being transcoded, could be with an icon or some information in the "details". 
At the moment the only way to check if the app is transcoding for me is to check the jellyfin logs from the android app to see if there is a transcoding log available or not.

**Comment by DJWhitegold** [comment]
Yeah I'd like that too. Perhaps even show the bit rate of the stream (if possible).

**Comment by strugee** [comment]
I'm curious what the use case here is?
